src/components/Select/SelectStyles.ts

```diff
@@ -0,0 +1,99 @@
+import { CSSObject } from '@emotion/serialize'
```

Comment:
Shouldn't this file be `.tsx`?
Let's keep all files as `.tsx` for sake of simplicity. It has no downsides, only upsides.

Reply:
OK, why not. I'll change it.